README.md

```diff
@@ -54,6 +55,21 @@ storage.csv."https://raw.githubusercontent.com/snowlift/trino-storage/master/src
 (2 rows)
 ```
 
+
+`ssv` plugin extract each line with `;` delimiter. Currently first line is used as column names.
```

Comment:
Took me a moment to figure out the first `s` stands for semicolon :D

Honestly, if this format is not very common, I'd rather add support for this in the table function. Maybe as a parameter, to make the csv reader configurable with custom separators, escape chars etc.

Reply:
Table functions are the first iteration, so we can discuss their design (naming). This would be a good opportunity to improve this API.

Reply:
It's fairly common in europe, in locales where `,` is used as a decimal separator. (Yes, you could argue that people should use escapes instead. You could also argue that people should use a less shit format than CSV in the first place. Sadly we're not in that world...)

Reply:
`ssv` is fine for now. I just wonder that we might not need a separate example in the readme.

Reply:
Or somehow combine all three sections for csv, tsv and ssv.

Reply:
Merged impls: fb5c26be34d359faf57217bb84cf2dff60181922

Reply:
How about something like https://gist.github.com/nightkr/cce2a6c85af7fefc718a18b52dda1955?

Reply:
Looks great, apply these changes here (squashed) and I'll merge this. Thanks!

Reply:
2c956702d8b3a3cb3980c615b1f1552147552b46

Reply:
Oh shit I'm dumb, uploaded it to the wrong branch, sec...

Reply:
2d06d338bed9297c5e95d770988145fa7aa0e72b should be the correct one!